internal/services/zero_trust_tunnel_cloudflared_virtual_network/resource_test.go

```diff
@@ -129,6 +156,52 @@ func TestAccCloudflareTunnelVirtualNetwork_Basic(t *testing.T) {
 	})
 }
 
+func TestAccCloudflareTunnelVirtualNetwork_Minimal(t *testing.T) {
```

Comment:
what is the difference between this test and TestAccCloudflareTunnelVirtualNetwork_Basic?

Reply:
Basic tests the full feature set with updates, while Minimal tests the bare minimum configuration with defaults.